content/news/2015/08/2015-08-20-a-six-month-update-on-how-weve-been-using-data-and-how-it-benefits-all-americans.md

```diff
@@ -7,7 +7,6 @@ summary: 'Memorandum: A Six Month Update on How We’ve Been Using Data, and How
 authors:
   - dr-dj-patil
 topics:
-  - data
   - Commerce
```

Comment:
```suggestion
  - Commerce
  - user-experience
```

Reply:
This is already archived; it has an `expirydate`, and is not in site's search results. 
 Would remove this from the pull request.